learn/react-native/integrating-crashlytics.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Integrating Crashlytics"
+id: "integrating-crashlytics"
+sidebar_label: "Crashlytics"
+---
+---
+import crashLogs from '/learn/assets/crash-logs-attributes-error-reports.png';
+
+Crashlytics is a crash reporting and analysis tool provided by Firebase. Crashlytics helps developers track and understand crashes that occur in their mobile applications. It provides insights into the causes of crashes and allows developers to take action to improve the stability of their apps, which essentially contributes to higher user satisfaction and increased app success.
+
+Crashlytics can be employed for the following reasons:
+
+- Crash Reporting
+- Error Insights
+- Real time Alerts
+- Custom Logs
+
+## Using Firebase Crashlytics
+To send crash logs to Crashlytics, you need to integrate the Crashlytics SDK into your Wavemaker application's codebase.
+If you don't have a Firebase project, create one on the Firebase Console: [console.firebase.google.com](https://console.firebase.google.com/)
+Once you have created a Firebase project, you can follow the steps below to integrate Crashlytics into a WaveMaker application.
+
+### Adding Crashlytics Plugin to an App
+
+Crashlytics plugins can be installed in a few steps in a WaveMaker application. Please refer to this [page](https://docs.wavemaker.com/learn/react-native/third-party-expo-plugins#expo)
+on how to install the plugin.
+
+
+1. Install and setup the app module.
```

Comment:
i don't understand this, how can WM app developer run npm command? The app he is developing is on WM cloud.